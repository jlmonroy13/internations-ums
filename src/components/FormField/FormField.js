import React from 'react';
import PropTypes from 'prop-types';

function FormField({ id, ...restProps }) {
  return (
    <div className="form__field">
      <label className="sr-only" htmlFor={id} />
      <input className="form__input" id={id} {...restProps} />
    </div>
  );
}

FormField.propTypes = {
  id: PropTypes.string.isRequired,
};

export default FormField;
