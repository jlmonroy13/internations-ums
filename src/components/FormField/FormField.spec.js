import React from 'react';
import renderer from 'react-test-renderer';
import FormField from './FormField';

describe('<FormField />', () => {
  it('should render the form field component', () => {
    const wrapper = renderer.create(<FormField id="" />);
    expect(wrapper).toMatchSnapshot();
  });
});
