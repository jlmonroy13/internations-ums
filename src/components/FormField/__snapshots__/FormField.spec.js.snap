// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<FormField /> should render the form field component 1`] = `
<div
  className="form__field"
>
  <label
    className="sr-only"
    htmlFor=""
  />
  <input
    className="form__input"
    id=""
  />
</div>
`;
